src/EFCore.MySql/Infrastructure/Internal/MySqlOptionsExtension.cs

```diff
@@ -73,6 +73,12 @@ protected override RelationalOptionsExtension Clone()
         /// </summary>
         public bool NoBackslashEscapes { get; private set; }
 
+        /// <summary>
+        ///     This API supports the Entity Framework Core infrastructure and is not intended to be used
+        ///     directly from your code. This API may change or be removed in future releases.
+        /// </summary>
+        public bool UpdateSqlModeOnOpen { get; private set; }
```

Comment:
This needs to be added to the `MySqlOptionsExtension([NotNull] MySqlOptionsExtension copyFrom)` constructor (that clones the instance).

Reply:
Good catch!